src/main/java/omero/cmd/admin/UpdateSessionTimeoutRequestI.java

```diff
@@ -143,6 +153,16 @@ protected Session updateSession() {
         }
 
         boolean isAdmin = current.getCurrentEventContext().isCurrentUserAdmin();
+        if (!isAdmin && maxUserTimeToLive != 0 && timeToLive != null
```

Comment:
negative value could be technically set so maybe ``maxUserTimeToLive > 0`` will be better

Reply:
A negative value is undefined. We can add checks for that here but would also need to handle those cases for the base values as well. IMHO well outside the scope of what this PR is trying to do.

Reply:
understood